Workspaces/Inspector/InspectorWorkspace.cs

```diff
@@ -26,6 +27,8 @@ public class InspectorWorkspace : Workspace, ISelectionChanged
 
 	bool m_IsLocked;
 
+	bool m_BlockUndoPostProcess;
```

Comment:
Not a fan of this approach. I'd say that we should likely subscribe and unsubscribe as needed, but I'm sure I'll understand more when we discuss this live.